tests/core/fmf-id/data/test-with-valid-ref.fmf

```diff
@@ -3,10 +3,12 @@ test: /bin/false
 require:
   - ref: branch-or-tag-ref
     type: library
+    path: /example
 
   # commit, an obviously hexadecimal one
   - ref: 8deadbeaf8
     type: library
+    path: /example
```

Comment:
Please use some url here instead.

Reply:
Updated to `url: http://localhost`, let me know if any other specific URL preferred.